aim/storage/locking.py

```diff
@@ -141,6 +141,12 @@ def AutoFileLock(
         A timeout of 0 means, that there is exactly one attempt to acquire the
         file lock.
     """
+    from aim.sdk.configs import AIM_USE_FILELOCK_ONLY
+
+    if os.environ.get(AIM_USE_FILELOCK_ONLY):
```

Comment:
We better move this after line 152. It will let us add a warning like a `Detected filesystem of type {fs_type}. Using filelock only because FORCE_USE_FILELOCK is set` which is good thing for debugging purposes.

Some users may wrongly think that they filesystem supports `UnixFileLock`s but eventually run into a problem. In that case, we'll see this warning message which will hint us about the problematic setup.